src/tokens/mainnet.json

```diff
@@ -659,6 +659,19 @@
         "0x6969696969696969696969696969696969696969"
       ]
     },
+    {
+      "address": "0x54c603173CA92A42B81f8838d705fd3E0f98D5D4",
```

Comment:
this address points to a different stake token:
https://berascan.com/address/0x54c603173CA92A42B81f8838d705fd3E0f98D5D4#readContract